_download-meta/openttd-releases.md

```diff
@@ -2,9 +2,13 @@
 folder_old_infrastructure: releases
 ---
 
-For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
-There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
-These can be installed automatically by the Windows and OS/2 installers.
-Please refer to the [readme](https://proxy.binaries.openttd.org/openttd-releases/@@version@@/README.md) for more information.
+OpenTTD requires one version of base graphics; available for installation with the installer are:
+* [OpenGFX (graphics)](../opengfx-releases/latest.html)
+* Transport Tycoon Deluxe data files can be installed if you own the Transport Tycoon Deluxe CD
```

Comment:
"Alternatively, the original Transport Tycoon Deluxe data files can be used as base graphics set. You must own a version of Transport Tycoon Deluxe, they cannot be downloaded."